DisplaySettings/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.3] - 2022-08-25
+### Fixed
+- Fixed std:: RDKTV-19091 : WpeFramework crash fix
```

Comment:
Don't mention JIRA tickets. Pls also remove RDKTV-18555 from the previous comment.